Source/Softswitch/src/softswitch_common.cpp

```diff
@@ -506,12 +514,13 @@ void softswitch_onReceive(ThreadCtxt_t* ThreadContext, volatile void* recv_buf)
     }
     else
     {
-        handler_log(5, "dIDX OOR %d", devAdr);
+        //handler_log(5, "dIDX OOR %d", devAdr);
```

Comment:
I'm all for keeping comments around if they're useful, but an explanation is helpful.

Reply:
Addressed in e1b5924dde536e02ebf62243cd62ee007173e730